Source/WebCore/rendering/style/RenderStyleConstants.h

```diff
@@ -1216,6 +1216,12 @@ enum class ContainIntrinsicSizeType : uint8_t {
     AutoAndLength
 };
 
+enum class ContentVisibility : uint8_t {
+    Visible,
+    Auto,
+    Hidden,
+};
+
```

Comment:
That is a nice style too, I noticed LazyImageLoadState enum class in ImageLoader does that. But somebody needs to do it for the whole RenderStyleConstants.h file or it would look odd IMHO.